src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -2435,19 +2435,23 @@ static void sub_6CBCE2(
         int32_t baseZ = (originZ + trackBlock->z) >> 3;
         int32_t clearanceZ = ((trackBlock->var_07 + RideData5[ride->type].clearance_height) >> 3) + baseZ + 4;
 
-        auto tileCoords = TileCoordsXY{ coords };
+        auto centreTileCoords = TileCoordsXY{ coords };
+        auto eastTileCoords = centreTileCoords + TileDirectionDelta[2];
```

Comment:
Use the named direction constants.

Reply:
I imagined there might have been some, but hadn't found them, thanks for the heads-up!